### ISSUE TITLE
FE Select custom polygons

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#211]
## Description
See Epic: #197 

## Acceptance criteria
* Select different polygons (shapes, etc)
* Visualize them (e.g.: border)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Corgam** [comment]
Done during the last sprints. Moved to the "Done" column during the sprint meeting by POs. Closing.
